server/src/main/scala/actors/DrtSystem.scala

```diff
@@ -62,7 +62,11 @@ trait DrtSystemInterface extends UserRoleProviderLike {
 }
 
 object DrtStaticParameters {
-  val expireAfterMillis: MillisSinceEpoch = 2 * oneDayMillis
+  val fortyEightHoursMillis: MillisSinceEpoch = 2 * oneDayMillis
```

Comment:
I'm not sure about this rename - the previous one left some leeway for us to change the time to expiry, however this one is pretty specific about what it should be. I'm in two minds though - perhaps it's better to be specific and we can rename the variable if we want to change the time limit.